internal/storage/writer/writer.go

```diff
@@ -112,7 +113,29 @@ func newWriter(config *config.Compaction) (flush.Writer, error) {
 
 	// Configure Talaria writer if present
 	if config.Talaria != nil {
-		w, err := talaria.New(config.Talaria.Endpoint, config.Talaria.DialTimeout, config.Talaria.CircuitTimeout, config.Talaria.MaxConcurrent, config.Talaria.ErrorPercentThreshold, config.Talaria.NonBlocking)
+		var defaultDialTimeout time.Duration = 5
+		var defaultCircuitTimeout time.Duration = 5
+		var defaultMaxConcurrent int = hystrix.DefaultMaxConcurrent
```

Comment:
Didn't use const for all default values because can't assign hystrix variable to const